Shoko.Server/Mappings/VideoLocal_PlaceMap.cs

```diff
@@ -15,5 +15,7 @@ public VideoLocal_PlaceMap()
         Map(x => x.FilePath).Not.Nullable();
         Map(x => x.ImportFolderID).Not.Nullable();
         Map(x => x.ImportFolderType).Not.Nullable();
+        Map(x => x.AllowAutoRelocation).Not.Nullable();
```

Comment:
Why? Per file overrides on this seem completely unnecessary

Reply:
To prevent it from being automagically moved, e.g. you place a copy at one location for it to stay there, and not be moved away by shoko. If we had a better renaming system that allowed the user to specify _all_ the paths you would want for a file then I don't it will be needed, no. But until then then I have a personal need for this per-file-location setting to exist.

Reply:
An good example is for Toriko EP1 / One Piece EP420(?), where _I_ would want to place a hard copy in both directories, and not have shoko touch them.

Reply:
Well it won't get moved if it's in a destination. For predictability and reproducibility, you want your script to give it exactly one place it should go.

Reply:
Also, I think i forgot to expose a way to toggle the per-file location settings (allow auto relocate and allow auto delete) in the api…

Reply:
That's a fair example, I guess. It's a very niche feature.

Reply:
This sounds a bit like over-engineering an issue that isn't really there.